substrate/frame/multisig/src/lib.rs

```diff
@@ -469,29 +503,23 @@ pub mod pallet {
 			call_hash: [u8; 32],
 		) -> DispatchResult {
 			let who = ensure_signed(origin)?;
-			ensure!(threshold >= 2, Error::<T>::MinimumThreshold);
-			let max_sigs = T::MaxSignatories::get() as usize;
-			ensure!(!other_signatories.is_empty(), Error::<T>::TooFewSignatories);
-			ensure!(other_signatories.len() < max_sigs, Error::<T>::TooManySignatories);
-			let signatories = Self::ensure_sorted_and_insert(other_signatories, who.clone())?;
-
-			let id = Self::multi_account_id(&signatories, threshold);
-
-			let m = <Multisigs<T>>::get(&id, call_hash).ok_or(Error::<T>::NotFound)?;
-			ensure!(m.when == timepoint, Error::<T>::WrongTimepoint);
-			ensure!(m.depositor == who, Error::<T>::NotOwner);
-
-			let err_amount = T::Currency::unreserve(&m.depositor, m.deposit);
-			debug_assert!(err_amount.is_zero());
-			<Multisigs<T>>::remove(&id, &call_hash);
+			Self::cancel(who, threshold, other_signatories, Some(timepoint), call_hash)
+		}
 
-			Self::deposit_event(Event::MultisigCancelled {
-				cancelling: who,
-				timepoint,
-				multisig: id,
-				call_hash,
-			});
-			Ok(())
+		/// Same as [`Pallet::cancel_as_multi`], but without a timepoint.
+		///
+		/// If the multisig is created with a `timepoint`, use [`Pallet::cancel_as_multi`],
```

Comment:
Why would the timepoint not be passed as `Option`?

Reply:
https://github.com/paritytech/polkadot-sdk/pull/2735/files#diff-a99723ce864d6d9e58c74c4a3246a8c9ba573564258190a799d65496537b8880R483-R484

I'd gladly turn it to `Option`, but then it would be a breaking change?

Reply:
is there a consensus on this? Are we introducing a breaking change, or keeping things as is?

Reply:
Okay new function is fine.